app/src/main/res/values/strings.xml

```diff
@@ -182,6 +182,9 @@
     <string name="create_faq">Create FAQ</string>
     <string name="question">Question</string>
     <string name="answer">Answer</string>
+    <string name="use_date_time_picker">Use Picker</string>
+    <string name="event_start_time">Starts at:</string>
+    <string name="event_end_time">Ends at:</string>
```

Comment:
Name should be `starts_at` `ends_at`

Reply:
I have removed them as we can pass those too, using binding. And currently we're using `sale_start_time` and `sale_end_time` in `create_event_form.xml`.